Dockerfile

```diff
@@ -21,11 +21,17 @@ USER root
 ADD . /che-operator
 WORKDIR /che-operator
 
+# build operator
 RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; elif [[ ${ARCH} == "aarch64" ]]; then export ARCH="arm64"; fi && \
     export MOCK_API=true && \
     go test -mod=vendor -v ./... && \
     GOOS=linux GOARCH=${ARCH} CGO_ENABLED=0 go build -mod=vendor -o /tmp/che-operator/che-operator cmd/manager/main.go
 
+# download devworkspace-operator templates
+RUN curl -L https://api.github.com/repos/devfile/devworkspace-operator/zipball/master > /tmp/devworkspace-operator.zip && \
```

Comment:
I think devworkspace operator version should be defined as argument.

Reply:
Right now there are only `master` and `v1.0.0-alpha1`
I can define env variable in the Dockerfile.

Reply:
it's just because release lack. But soon releases will be with Che cadense.
If you meant argument, that +1
If you meant environment variable - please elaborate more

Reply:
`ENV DEV_WORKSPACE_CONTROLLER_VERSION="master"`
When che-operator is released the correct version will be set

Reply:
Why it's env var but not argument? Like here https://github.com/eclipse/che/blob/master/dockerfiles/che/Dockerfile#L11

Reply:
ok. make sense